Dockerfile

```diff
@@ -22,20 +22,20 @@ FROM node:6-alpine
 LABEL xo-server=5.18.3 \
          xo-web=5.18.2
 
-ENV USER=node \
-    USER_HOME=/home/node \
+ENV USER=root \
```

Comment:
this configuration set the environment variable `USER` you can overwrite it on your `docker-compose.yml` or change this, like do in the build file, and the application should run with that user. So a simple `USER=root` should be enough to run the application with the root user.